src/display/editor/tools.js

```diff
@@ -919,6 +919,14 @@ class AnnotationEditorUIManager {
    */
   paste(event) {
     event.preventDefault();
+    const { clipboardData } = event;
+    for (const item of clipboardData.items) {
+      for (const editorType of this.#editorTypes) {
+        if (editorType.isHandlingMimeForPasting(item.type)) {
+          editorType.paste(item, this.currentLayer);
```

Comment:
Should we perhaps return early here, after having handled the paste-event, since it's not entirely clear to me if the code below makes sense then?